src/api/spec/models/package_spec.rb

```diff
@@ -401,7 +401,8 @@
   describe '#jobhistory_list' do
     let(:backend_url) { "#{CONFIG['source_url']}/build/#{home_project}/openSUSE_Tumbleweed/x86_64/_jobhistory?limit=100&package=#{package}" }
 
-    subject { package.jobhistory_list(home_project, 'openSUSE_Tumbleweed', 'x86_64') }
+    # the package name is needed because it could be a multibuild
+    subject { package.jobhistory_list(home_project, 'openSUSE_Tumbleweed', 'x86_64', package.name) }
```

Comment:
Looks like this broke with your previous change ("[api] package as param instead of an options hash"). Please squash them together

Reply:
This is still not squashed:-)